middleware/http/sentinel/logger.go

```diff
@@ -0,0 +1,46 @@
+package sentinel
+
+import (
+	"github.com/alibaba/sentinel-golang/logging"
+	"github.com/dapr/dapr/pkg/logger"
```

Comment:
This has moved to `"github.com/dapr/kit/logger"`

Reply:
fixed.